src/app/Http/Controllers/Operations/Concerns/HasForm.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Backpack\CRUD\app\Http\Controllers\Operations\Concerns;
+
+use Illuminate\Support\Facades\Route;
+use Illuminate\Support\Str;
+
+trait HasForm
+{
+    protected function formRoutes(string $operationName, bool $routesHaveIdSegment, string $segment, string $routeName, string $controller): void
+    {
+        $secondSegment = $routesHaveIdSegment ? '/{id}/' : '/';
+
+        Route::get($segment.$secondSegment.Str::of($operationName)->kebab(), [
+            'as'        => $routeName.'.get'.$operationName.'Form',
+            'uses'      => $controller.'@get'.$operationName.'Form',
+            'operation' => $operationName,
+        ]);
+        Route::post($segment.$secondSegment.Str::of($operationName)->kebab(), [
+            'as'        => $routeName.'.post'.$operationName.'Form',
+            'uses'      => $controller.'@post'.$operationName.'Form',
+            'operation' => $operationName,
+        ]);
+    }
+
+    /**
+     * Method to handle the GET request and display the View with a Backpack form.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Contracts\View\View
+     */
+    public function getForm($id = null)
```

Comment:
Once HasForm is used, this will be a method on the CrudController... so maybe the name is too general? Might conflict with custom methods.